
From: Albert Herranz <albert_herranz@yahoo.es>

The following patch places KEXEC Kconfig ppc option in a more convenient
menu, like in the i386 tree.  It seems that it got again misplaced (in the
current -mm tree it appears within the IBM 40x options menu).

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/ppc/Kconfig |   40 ++++++++++++++++++++--------------------
 1 files changed, 20 insertions(+), 20 deletions(-)

diff -puN arch/ppc/Kconfig~kexec-ppc-kexec-kconfig-misplacement arch/ppc/Kconfig
--- 25/arch/ppc/Kconfig~kexec-ppc-kexec-kconfig-misplacement	Mon Aug 30 16:20:09 2004
+++ 25-akpm/arch/ppc/Kconfig	Mon Aug 30 16:20:09 2004
@@ -245,6 +245,26 @@ config NOT_COHERENT_CACHE
 
 source "drivers/perfctr/Kconfig"
 
+config KEXEC
+	bool "kexec system call (EXPERIMENTAL)"
+	depends on EXPERIMENTAL
+	help
+	  kexec is a system call that implements the ability to shutdown your
+	  current kernel, and to start another kernel.  It is like a reboot
+	  but it is indepedent of the system firmware.   And like a reboot
+	  you can start any kernel with it, not just Linux.
+
+	  The name comes from the similiarity to the exec system call.
+
+	  It is an ongoing process to be certain the hardware in a machine
+	  is properly shutdown, so do not be surprised if this code does not
+	  initially work for you.  It may help to enable device hotplugging
+	  support.  As of this writing the exact hardware interface is
+	  strongly in flux, so no good recommendation can be made.
+
+	  In the GameCube implementation, kexec allows you to load and
+	  run DOL files, including kernel and homebrew DOLs.
+
 endmenu
 
 menu "Platform options"
@@ -1222,26 +1242,6 @@ config SERIAL_SICC_CONSOLE
 	depends on SERIAL_SICC && UART0_TTYS1
 	default y
 
-config KEXEC
-	bool "kexec system call (EXPERIMENTAL)"
-	depends on EXPERIMENTAL
-	help
-	  kexec is a system call that implements the ability to shutdown your
-	  current kernel, and to start another kernel.  It is like a reboot
-	  but it is indepedent of the system firmware.   And like a reboot
-	  you can start any kernel with it, not just Linux.
-
-	  The name comes from the similiarity to the exec system call.
-
-	  It is an ongoing process to be certain the hardware in a machine
-	  is properly shutdown, so do not be surprised if this code does not
-	  initially work for you.  It may help to enable device hotplugging
-	  support.  As of this writing the exact hardware interface is
-	  strongly in flux, so no good recommendation can be made.
-
-	  In the GAMECUBE implementation, kexec allows you to load and
-	  run DOL files, including kernel and homebrew DOLs.
-
 endmenu
 
 source "lib/Kconfig"
_
